active/XamlServiceProvider/XamlServiceProviderApi.md

```diff
@@ -0,0 +1,251 @@
+
+# Background
+
+_This spec adds a serviceProvider to the existing [MarkupExtension](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Markup.MarkupExtension) class. ([Proposal](https://github.com/microsoft/microsoft-ui-xaml/issues/741))_
+
+Xaml markup has a [markup extension](https://docs.microsoft.com/en-us/dotnet/desktop-wpf/xaml-services/markup-extensions-overview) feature which is a builder pattern, similar to a .Net StringBuilder. It lets you create, modify, and then retrieve a value.
+
+When you write a custom markup extension (see the Description section below), you override the `ProvideValue` method to return the built value. The ProvideValue method is different in  Xaml (WinUI) than it is in WPF, because WPF passes an extra [IServiceProvider](http://msdn.microsoft.com/library/System.IServiceProvider) parameter:

```

Comment:
Referring to ' Xaml (WinUI)' gives the impression that you're thinking of WinUI as a replacement to XAML.
XAML is a markup language used by WinUI and WPF (& Xamarin.Forms, and more.)
Using "XAML" to mean "WinUI" limits the scope of WinUI and is potentially confusing.